cli/azd/pkg/account/manager_test.go

```diff
@@ -395,7 +395,7 @@ func Test_HasDefaults(t *testing.T) {
 		)
 		require.NoError(t, err)
 
-		value := manager.HasDefaults()
+		value, _ := manager.HasDefaults(*mockContext.Context)
```

Comment:
It would be nice if we could write a test that locks in the new behavior of HasDefaults where there can be defaults but they are invalid.  I think it would be pretty easy for us to write this test given the mocking we can do today (I think we just have to mock the get subscription call to return a 404?)

Reply:
done